lib/views/create-bug.js

```diff
@@ -220,8 +211,8 @@ function submitBug() {
     function createAttachments() {
       if (!bug.attachments.length) {
         localforage.setItem(KEY, null).then(function() {
-          dialog.close();
           app.page('/bug/' + result.id);
```

Comment:
Sorry no, was a small issue I noticed while writing the patch, it doesnt actually make any difference